compress_jointcal_cfht_test_data.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+"""Zero out the imageHDUs in the calexps, and then gzip the files."""
+from __future__ import absolute_import, division, print_function
+
+from astropy.io import fits
+import glob
+import subprocess
+
+files = glob.glob('cfht/calexp/06AL01/D3/2006-0*/r/calexp*.fits')
+for f in files:
+    data = fits.open(f)
+    print('processing:', f)
+    for i in range(len(data)):
+        if isinstance(data[i], fits.ImageHDU):
+            data[i].data[:] = 0
```

Comment:
I agree that it's a limitation, but I think for the purposes of this testing data it's ok. Once this gets lifted up to validate_drp, we'll have the full images available to work with.